aim/tests/unit/tools/cli/test_manager.py

```diff
@@ -389,6 +390,52 @@ def test_sync_state_recover(self):
             # Items are updated
             self.assertEqual(4, up.call_count)
 
+    def test_tag_list_command(self):
+        self.mgr.create(self.ctx, resource.Tenant(name='tn1'))
+        mo_dict = {'tagInst': {'attributes':
+                   {'dn': 'uni/tn-tn1/tag-openstack_aid'}}}
+        mock_mo = mock.MagicMock()
+        mock_mo.__getitem__.side_effect = mo_dict.__getitem__
+        MO_LIST = [mock_mo]
+        apic_manager_object = mock.Mock()
+
+        def get_mgr():
+            return apic_manager_object
+        apic_manager_object.apic.list_mo = mock.Mock(
+            return_value=MO_LIST)
+        apic_manager_object.apic.transaction = mock.MagicMock()
+        with mock.patch('aim.tools.cli.commands.infra.get_apic_manager',
+                        side_effect=get_mgr):
+            self.run_command('infra tag-list -t openstack_aid')
+        # REVISIT: Find a better way of creating the verify call object.
+        # I did this because I couldn't find another way of creating an
+        # equivalent call object
+        verify_mo = mock.MagicMock()
```

Comment:
Maybe we can just create a mock for click.echo then verify that its getting called once with the parameter "uni/tn-tn1/tag-openstack_aid"?

Reply:
Will look into that.